./gradlew clean build fails on 2.0.0 and 2.1.0

on v1.0 I can successfully build:
```
⋊> ~/g/3/k/status-keycard-java on 8950e15 ⨯ ./gradlew clean build                                                                                                       

> Configure project :demo
Observed package id 'add-ons;addon-google_apis-google-19' in inconsistent location '/home/ligi/bin/android-sdk/add-ons/addon-google_apis-google-19-1' (Expected '/home/ligi/bin/android-sdk/add-ons/addon-google_apis-google-19')

> Task :demo:lint
Calling mockable JAR artifact transform to create file: /home/ligi/.gradle/caches/transforms-1/files-1.1/android.jar/50f20c2363d4276c27986dc1aff7a2e9/android.jar with input /home/ligi/bin/android-sdk/platforms/android-28/android.jar
Ran lint on variant release: 3 issues found
Ran lint on variant debug: 3 issues found
Wrote HTML report to file:///home/ligi/git/3rd/keycard/status-keycard-java/demo/build/reports/lint-results.html
Wrote XML report to file:///home/ligi/git/3rd/keycard/status-keycard-java/demo/build/reports/lint-results.xml

> Task :lib:lint
Ran lint on variant debug: 1 issues found
Ran lint on variant release: 1 issues found
Wrote HTML report to file:///home/ligi/git/3rd/keycard/status-keycard-java/lib/build/reports/lint-results.html
Wrote XML report to file:///home/ligi/git/3rd/keycard/status-keycard-java/lib/build/reports/lint-results.xml

BUILD SUCCESSFUL in 39s
113 actionable tasks: 112 executed, 1 up-to-date
⋊> ~/g/3/k/status-keycard-java on 8950e15
```
on 2.1.0 it fails with:

```
⋊> ~/g/3/k/status-keycard-java on 425d085 ⨯ ./gradlew clean build                                                                                                       10:37:09

> Task :android:javadoc
/home/ligi/git/3rd/keycard/status-keycard-java/android/src/main/java/im/status/keycard/android/NFCCardManager.java:8: error: package im.status.keycard.globalplatform does not exist
import im.status.keycard.globalplatform.Crypto;
                                       ^
/home/ligi/git/3rd/keycard/status-keycard-java/android/src/main/java/im/status/keycard/android/NFCCardManager.java:9: error: package im.status.keycard.io does not exist
import im.status.keycard.io.CardListener;
                           ^
/home/ligi/git/3rd/keycard/status-keycard-java/android/src/main/java/im/status/keycard/android/NFCCardManager.java:23: error: cannot find symbol
  private CardListener cardListener;
          ^
  symbol:   class CardListener
  location: class NFCCardManager
/home/ligi/git/3rd/keycard/status-keycard-java/android/src/main/java/im/status/keycard/android/NFCCardManager.java:118: error: cannot find symbol
  public void setCardListener(CardListener listener) {
                              ^
  symbol:   class CardListener
  location: class NFCCardManager
/home/ligi/git/3rd/keycard/status-keycard-java/android/src/main/java/im/status/keycard/android/NFCCardChannel.java:5: error: package im.status.keycard.io does not exist
import im.status.keycard.io.APDUCommand;
                           ^
/home/ligi/git/3rd/keycard/status-keycard-java/android/src/main/java/im/status/keycard/android/NFCCardChannel.java:6: error: package im.status.keycard.io does not exist
import im.status.keycard.io.APDUResponse;
                           ^
/home/ligi/git/3rd/keycard/status-keycard-java/android/src/main/java/im/status/keycard/android/NFCCardChannel.java:7: error: package im.status.keycard.io does not exist
import im.status.keycard.io.CardChannel;
                           ^
/home/ligi/git/3rd/keycard/status-keycard-java/android/src/main/java/im/status/keycard/android/NFCCardChannel.java:14: error: cannot find symbol
public class NFCCardChannel implements CardChannel {
                                       ^
  symbol: class CardChannel
/home/ligi/git/3rd/keycard/status-keycard-java/android/src/main/java/im/status/keycard/android/NFCCardChannel.java:24: error: cannot find symbol
  public APDUResponse send(APDUCommand cmd) throws IOException {
                           ^
  symbol:   class APDUCommand
  location: class NFCCardChannel
/home/ligi/git/3rd/keycard/status-keycard-java/android/src/main/java/im/status/keycard/android/NFCCardChannel.java:24: error: cannot find symbol
  public APDUResponse send(APDUCommand cmd) throws IOException {
         ^
  symbol:   class APDUResponse
  location: class NFCCardChannel
10 warnings

> Task :android:lint
Ran lint on variant debug: 0 issues found
Ran lint on variant release: 0 issues found
D8: Default interface methods are only supported starting with Android N (--min-api 24): int im.status.keycard.io.CardChannel.pairingPasswordPBKDF2IterationCount()

> Task :demo-android:transformClassesWithDexBuilderForDebug FAILED
com.android.builder.dexing.DexArchiveBuilderException: com.android.builder.dexing.DexArchiveBuilderException: Failed to process /home/ligi/git/3rd/keycard/status-keycard-java/lib/build/libs/lib.jar
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
        at java.util.concurrent.ForkJoinTask.getThrowableException(ForkJoinTask.java:593)
        at java.util.concurrent.ForkJoinTask.reportException(ForkJoinTask.java:677)
        at java.util.concurrent.ForkJoinTask.join(ForkJoinTask.java:720)
        at com.android.ide.common.internal.WaitableExecutor.waitForTasksWithQuickFail(WaitableExecutor.java:146)
        at com.android.build.gradle.internal.transforms.DexArchiveBuilderTransform.transform(DexArchiveBuilderTransform.java:405)
        at com.android.build.gradle.internal.pipeline.TransformTask$2.call(TransformTask.java:239)
        at com.android.build.gradle.internal.pipeline.TransformTask$2.call(TransformTask.java:235)
        at com.android.builder.profile.ThreadRecorder.record(ThreadRecorder.java:102)
        at com.android.build.gradle.internal.pipeline.TransformTask.transform(TransformTask.java:230)
        at sun.reflect.GeneratedMethodAccessor387.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:73)
        at org.gradle.api.internal.project.taskfactory.IncrementalTaskAction.doExecute(IncrementalTaskAction.java:50)
        at org.gradle.api.internal.project.taskfactory.StandardTaskAction.execute(StandardTaskAction.java:39)
        at org.gradle.api.internal.project.taskfactory.StandardTaskAction.execute(StandardTaskAction.java:26)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter$1.run(ExecuteActionsTaskExecuter.java:131)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:300)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:292)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:174)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:90)
        at org.gradle.internal.operations.DelegatingBuildOperationExecutor.run(DelegatingBuildOperationExecutor.java:31)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeAction(ExecuteActionsTaskExecuter.java:120)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:99)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:77)
        at org.gradle.api.internal.tasks.execution.OutputDirectoryCreatingTaskExecuter.execute(OutputDirectoryCreatingTaskExecuter.java:51)
        at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:59)
        at org.gradle.api.internal.tasks.execution.ResolveTaskOutputCachingStateExecuter.execute(ResolveTaskOutputCachingStateExecuter.java:54)
        at org.gradle.api.internal.tasks.execution.ValidatingTaskExecuter.execute(ValidatingTaskExecuter.java:59)
        at org.gradle.api.internal.tasks.execution.SkipEmptySourceFilesTaskExecuter.execute(SkipEmptySourceFilesTaskExecuter.java:101)
        at org.gradle.api.internal.tasks.execution.FinalizeInputFilePropertiesTaskExecuter.execute(FinalizeInputFilePropertiesTaskExecuter.java:44)
        at org.gradle.api.internal.tasks.execution.CleanupStaleOutputsExecuter.execute(CleanupStaleOutputsExecuter.java:91)
        at org.gradle.api.internal.tasks.execution.ResolveTaskArtifactStateTaskExecuter.execute(ResolveTaskArtifactStateTaskExecuter.java:62)
        at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:59)
        at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:54)
        at org.gradle.api.internal.tasks.execution.ExecuteAtMostOnceTaskExecuter.execute(ExecuteAtMostOnceTaskExecuter.java:43)
        at org.gradle.api.internal.tasks.execution.CatchExceptionTaskExecuter.execute(CatchExceptionTaskExecuter.java:34)
        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.run(EventFiringTaskExecuter.java:51)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:300)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:292)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:174)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:90)
        at org.gradle.internal.operations.DelegatingBuildOperationExecutor.run(DelegatingBuildOperationExecutor.java:31)
        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter.execute(EventFiringTaskExecuter.java:46)
        at org.gradle.execution.taskgraph.LocalTaskInfoExecutor.execute(LocalTaskInfoExecutor.java:42)
        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$BuildOperationAwareWorkItemExecutor.execute(DefaultTaskExecutionGraph.java:277)
        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$BuildOperationAwareWorkItemExecutor.execute(DefaultTaskExecutionGraph.java:262)
        at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$ExecutorWorker$1.execute(DefaultTaskPlanExecutor.java:135)
        at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$ExecutorWorker$1.execute(DefaultTaskPlanExecutor.java:130)
        at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$ExecutorWorker.execute(DefaultTaskPlanExecutor.java:200)
        at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$ExecutorWorker.executeWithWork(DefaultTaskPlanExecutor.java:191)
        at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$ExecutorWorker.run(DefaultTaskPlanExecutor.java:130)
        at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
        at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
        at java.lang.Thread.run(Thread.java:748)
Caused by: com.android.builder.dexing.DexArchiveBuilderException: Failed to process /home/ligi/git/3rd/keycard/status-keycard-java/lib/build/libs/lib.jar
        at com.android.build.gradle.internal.transforms.DexArchiveBuilderTransform.launchProcessing(DexArchiveBuilderTransform.java:900)
        at com.android.build.gradle.internal.transforms.DexArchiveBuilderTransform.lambda$convertToDexArchive$6(DexArchiveBuilderTransform.java:825)
        at java.util.concurrent.ForkJoinTask$AdaptedCallable.exec(ForkJoinTask.java:1424)
        at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
        at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
        at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
        at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
Caused by: com.android.builder.dexing.DexArchiveBuilderException: Error while dexing.
        at com.android.builder.dexing.D8DexArchiveBuilder.getExceptionToRethrow(D8DexArchiveBuilder.java:124)
        at com.android.builder.dexing.D8DexArchiveBuilder.convert(D8DexArchiveBuilder.java:101)
        at com.android.build.gradle.internal.transforms.DexArchiveBuilderTransform.launchProcessing(DexArchiveBuilderTransform.java:895)
        ... 6 more
Caused by: com.android.tools.r8.CompilationFailedException: Compilation failed to complete
        at com.android.tools.r8.utils.ExceptionUtils.withCompilationHandler(ExceptionUtils.java:65)
        at com.android.tools.r8.utils.ExceptionUtils.withD8CompilationHandler(ExceptionUtils.java:43)
        at com.android.tools.r8.D8.run(D8.java:90)
        at com.android.builder.dexing.D8DexArchiveBuilder.convert(D8DexArchiveBuilder.java:99)
        ... 7 more
Caused by: com.android.tools.r8.utils.AbortException: Error: Default interface methods are only supported starting with Android N (--min-api 24): int im.status.keycard.io.CardChannel.pairingPasswordPBKDF2IterationCount()
        at com.android.tools.r8.utils.Reporter.failIfPendingErrors(Reporter.java:116)
        at com.android.tools.r8.utils.Reporter.fatalError(Reporter.java:74)
        at com.android.tools.r8.utils.ExceptionUtils.withCompilationHandler(ExceptionUtils.java:59)
        ... 10 more


FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':demo-android:transformClassesWithDexBuilderForDebug'.
> com.android.build.api.transform.TransformException: com.android.builder.dexing.DexArchiveBuilderException: com.android.builder.dexing.DexArchiveBuilderException: Failed to process /home/ligi/git/3rd/keycard/status-keycard-java/lib/build/libs/lib.jar

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 8s
81 actionable tasks: 77 executed, 4 up-to-date

```

on 2.0.0 it fails with:

```
⋊> ~/g/3/k/status-keycard-java on 0d16541 ⨯ ./gradlew clean build                                                                                                       10:35:40

> Configure project :android
Observed package id 'add-ons;addon-google_apis-google-19' in inconsistent location '/home/ligi/bin/android-sdk/add-ons/addon-google_apis-google-19-1' (Expected '/home/ligi/bin/android-sdk/add-ons/addon-google_apis-google-19')

> Task :android:javadoc
/home/ligi/git/3rd/keycard/status-keycard-java/android/src/main/java/im/status/keycard/android/NFCCardManager.java:8: error: package im.status.keycard.globalplatform does not exist
import im.status.keycard.globalplatform.Crypto;
                                       ^
/home/ligi/git/3rd/keycard/status-keycard-java/android/src/main/java/im/status/keycard/android/NFCCardManager.java:9: error: package im.status.keycard.io does not exist
import im.status.keycard.io.CardListener;
                           ^
/home/ligi/git/3rd/keycard/status-keycard-java/android/src/main/java/im/status/keycard/android/NFCCardManager.java:23: error: cannot find symbol
  private CardListener cardListener;
          ^
  symbol:   class CardListener
  location: class NFCCardManager
/home/ligi/git/3rd/keycard/status-keycard-java/android/src/main/java/im/status/keycard/android/NFCCardManager.java:118: error: cannot find symbol
  public void setCardListener(CardListener listener) {
                              ^
  symbol:   class CardListener
  location: class NFCCardManager
/home/ligi/git/3rd/keycard/status-keycard-java/android/src/main/java/im/status/keycard/android/NFCCardChannel.java:5: error: package im.status.keycard.io does not exist
import im.status.keycard.io.APDUCommand;
                           ^
/home/ligi/git/3rd/keycard/status-keycard-java/android/src/main/java/im/status/keycard/android/NFCCardChannel.java:6: error: package im.status.keycard.io does not exist
import im.status.keycard.io.APDUResponse;
                           ^
/home/ligi/git/3rd/keycard/status-keycard-java/android/src/main/java/im/status/keycard/android/NFCCardChannel.java:7: error: package im.status.keycard.io does not exist
import im.status.keycard.io.CardChannel;
                           ^
/home/ligi/git/3rd/keycard/status-keycard-java/android/src/main/java/im/status/keycard/android/NFCCardChannel.java:14: error: cannot find symbol
public class NFCCardChannel implements CardChannel {
                                       ^
  symbol: class CardChannel
/home/ligi/git/3rd/keycard/status-keycard-java/android/src/main/java/im/status/keycard/android/NFCCardChannel.java:24: error: cannot find symbol
  public APDUResponse send(APDUCommand cmd) throws IOException {
                           ^
  symbol:   class APDUCommand
  location: class NFCCardChannel
/home/ligi/git/3rd/keycard/status-keycard-java/android/src/main/java/im/status/keycard/android/NFCCardChannel.java:24: error: cannot find symbol
  public APDUResponse send(APDUCommand cmd) throws IOException {
         ^
  symbol:   class APDUResponse
  location: class NFCCardChannel
10 warnings

> Task :android:lint
Ran lint on variant debug: 0 issues found
Ran lint on variant release: 0 issues found

> Task :demo-android:lint
Ran lint on variant release: 3 issues found
Ran lint on variant debug: 3 issues found
Wrote HTML report to file:///home/ligi/git/3rd/keycard/status-keycard-java/demo-android/build/reports/lint-results.html
Wrote XML report to file:///home/ligi/git/3rd/keycard/status-keycard-java/demo-android/build/reports/lint-results.xml

> Task :lib:javadoc FAILED
/home/ligi/git/3rd/keycard/status-keycard-java/lib/src/main/java/im/status/keycard/applet/BIP32KeyPair.java:123: warning: no @param for includePublic
  public byte[] toTLV(boolean includePublic) {
                ^
/home/ligi/git/3rd/keycard/status-keycard-java/lib/src/main/java/im/status/keycard/applet/CardDuplicator.java:62: warning: no @return
  public byte[] exportKey(CardChannel channel, Pairing pairing, String pin) throws IOException, APDUException {
                ^
/home/ligi/git/3rd/keycard/status-keycard-java/lib/src/main/java/im/status/keycard/applet/CardDuplicator.java:86: warning: no description for @param
   * @param channel
     ^
/home/ligi/git/3rd/keycard/status-keycard-java/lib/src/main/java/im/status/keycard/applet/Identifiers.java:31: warning: no @param for instanceIdx
  public static byte[] getKeycardInstanceAID(int instanceIdx) {
                       ^
/home/ligi/git/3rd/keycard/status-keycard-java/lib/src/main/java/im/status/keycard/applet/KeycardCommandSet.java:85: warning: no @return
  public Pairing getPairing() {
                 ^
/home/ligi/git/3rd/keycard/status-keycard-java/lib/src/main/java/im/status/keycard/applet/KeycardCommandSet.java:142: warning: no @param for pairingPassword
  public void autoPair(String pairingPassword) throws IOException {
              ^
/home/ligi/git/3rd/keycard/status-keycard-java/lib/src/main/java/im/status/keycard/applet/KeycardCommandSet.java:172: warning: no @param for sharedSecret
  public void autoPair(byte[] sharedSecret) throws IOException {
              ^
/home/ligi/git/3rd/keycard/status-keycard-java/lib/src/main/java/im/status/keycard/applet/KeycardCommandSet.java:188: warning: no @param for index
  public APDUResponse openSecureChannel(byte index, byte[] data) throws IOException {
                      ^
/home/ligi/git/3rd/keycard/status-keycard-java/lib/src/main/java/im/status/keycard/applet/KeycardCommandSet.java:188: warning: no @param for data
  public APDUResponse openSecureChannel(byte index, byte[] data) throws IOException {
                      ^
/home/ligi/git/3rd/keycard/status-keycard-java/lib/src/main/java/im/status/keycard/applet/KeycardCommandSet.java:188: warning: no @return
  public APDUResponse openSecureChannel(byte index, byte[] data) throws IOException {
                      ^
/home/ligi/git/3rd/keycard/status-keycard-java/lib/src/main/java/im/status/keycard/applet/KeycardCommandSet.java:188: warning: no @throws for java.io.IOException
  public APDUResponse openSecureChannel(byte index, byte[] data) throws IOException {
                      ^
/home/ligi/git/3rd/keycard/status-keycard-java/lib/src/main/java/im/status/keycard/applet/KeycardCommandSet.java:195: warning: no @return
  public APDUResponse mutuallyAuthenticate() throws IOException {
                      ^
/home/ligi/git/3rd/keycard/status-keycard-java/lib/src/main/java/im/status/keycard/applet/KeycardCommandSet.java:195: warning: no @throws for java.io.IOException
  public APDUResponse mutuallyAuthenticate() throws IOException {
                      ^
/home/ligi/git/3rd/keycard/status-keycard-java/lib/src/main/java/im/status/keycard/applet/KeycardCommandSet.java:202: warning: no @param for data
  public APDUResponse mutuallyAuthenticate(byte[] data) throws IOException {
                      ^
/home/ligi/git/3rd/keycard/status-keycard-java/lib/src/main/java/im/status/keycard/applet/KeycardCommandSet.java:202: warning: no @return
  public APDUResponse mutuallyAuthenticate(byte[] data) throws IOException {
                      ^
/home/ligi/git/3rd/keycard/status-keycard-java/lib/src/main/java/im/status/keycard/applet/KeycardCommandSet.java:202: warning: no @throws for java.io.IOException
  public APDUResponse mutuallyAuthenticate(byte[] data) throws IOException {
                      ^
/home/ligi/git/3rd/keycard/status-keycard-java/lib/src/main/java/im/status/keycard/applet/KeycardCommandSet.java:209: warning: no @param for p1
  public APDUResponse pair(byte p1, byte[] data) throws IOException {
                      ^
/home/ligi/git/3rd/keycard/status-keycard-java/lib/src/main/java/im/status/keycard/applet/KeycardCommandSet.java:209: warning: no @param for data
  public APDUResponse pair(byte p1, byte[] data) throws IOException {
                      ^
/home/ligi/git/3rd/keycard/status-keycard-java/lib/src/main/java/im/status/keycard/applet/KeycardCommandSet.java:209: warning: no @return
  public APDUResponse pair(byte p1, byte[] data) throws IOException {
                      ^
/home/ligi/git/3rd/keycard/status-keycard-java/lib/src/main/java/im/status/keycard/applet/KeycardCommandSet.java:209: warning: no @throws for java.io.IOException
  public APDUResponse pair(byte p1, byte[] data) throws IOException {
                      ^
/home/ligi/git/3rd/keycard/status-keycard-java/lib/src/main/java/im/status/keycard/applet/KeycardCommandSet.java:216: warning: no @param for p1
  public APDUResponse unpair(byte p1) throws IOException {
                      ^
/home/ligi/git/3rd/keycard/status-keycard-java/lib/src/main/java/im/status/keycard/applet/KeycardCommandSet.java:216: warning: no @return
  public APDUResponse unpair(byte p1) throws IOException {
                      ^
/home/ligi/git/3rd/keycard/status-keycard-java/lib/src/main/java/im/status/keycard/applet/KeycardCommandSet.java:216: warning: no @throws for java.io.IOException
  public APDUResponse unpair(byte p1) throws IOException {
                      ^
/home/ligi/git/3rd/keycard/status-keycard-java/lib/src/main/java/im/status/keycard/applet/KeycardCommandSet.java:223: warning: no @throws for java.io.IOException
  public void unpairOthers() throws IOException, APDUException {
              ^
/home/ligi/git/3rd/keycard/status-keycard-java/lib/src/main/java/im/status/keycard/applet/KeycardCommandSet.java:223: warning: no @throws for im.status.keycard.io.APDUException
  public void unpairOthers() throws IOException, APDUException {
              ^
/home/ligi/git/3rd/keycard/status-keycard-java/lib/src/main/java/im/status/keycard/applet/KeycardCommandSet.java:299: warning: no @param for puk
  public APDUResponse unblockPIN(String puk, String newPin) throws IOException {
                      ^
/home/ligi/git/3rd/keycard/status-keycard-java/lib/src/main/java/im/status/keycard/applet/KeycardCommandSet.java:299: warning: no @param for newPin
  public APDUResponse unblockPIN(String puk, String newPin) throws IOException {
                      ^
/home/ligi/git/3rd/keycard/status-keycard-java/lib/src/main/java/im/status/keycard/applet/KeycardCommandSet.java:571: warning: no @param for source
  public APDUResponse exportKey(byte[] keyPath, int source, boolean makeCurrent, boolean publicOnly) throws IOException {
                      ^
/home/ligi/git/3rd/keycard/status-keycard-java/lib/src/main/java/im/status/keycard/applet/Mnemonic.java:33: warning: no description for @param
   * @param wordlist
     ^
/home/ligi/git/3rd/keycard/status-keycard-java/lib/src/main/java/im/status/keycard/applet/Mnemonic.java:126: warning: no @return
  public BIP32KeyPair toBIP32KeyPair() {
                      ^
/home/ligi/git/3rd/keycard/status-keycard-java/lib/src/main/java/im/status/keycard/applet/SecureChannelSession.java:181: warning: no @param for sharedSecret
  public void autoPair(CardChannel apduChannel, byte[] sharedSecret) throws IOException {
              ^
/home/ligi/git/3rd/keycard/status-keycard-java/lib/src/main/java/im/status/keycard/applet/SecureChannelSession.java:313: warning: no @throws for im.status.keycard.io.APDUException
  public void unpairOthers(CardChannel apduChannel) throws IOException, APDUException {
              ^
/home/ligi/git/3rd/keycard/status-keycard-java/lib/src/main/java/im/status/keycard/globalplatform/GlobalPlatformCommandSet.java:73: warning: no @throws for im.status.keycard.io.APDUException
  public APDUResponse initializeUpdate(byte[] hostChallenge) throws IOException, APDUException {
                      ^
/home/ligi/git/3rd/keycard/status-keycard-java/lib/src/main/java/im/status/keycard/globalplatform/GlobalPlatformCommandSet.java:258: error: bad HTML entity
   * Sends an INSTALL [for Install & Make Selectable] command. Use only if not installing applets part of the Keycard
                                   ^
/home/ligi/git/3rd/keycard/status-keycard-java/lib/src/main/java/im/status/keycard/globalplatform/Load.java:32: warning: no description for @throws
   * @throws FileNotFoundException
     ^
/home/ligi/git/3rd/keycard/status-keycard-java/lib/src/main/java/im/status/keycard/globalplatform/Load.java:33: warning: no description for @throws
   * @throws IOException
     ^
/home/ligi/git/3rd/keycard/status-keycard-java/lib/src/main/java/im/status/keycard/io/APDUCommand.java:56: warning: no @throws for java.io.IOException
  public byte[] serialize() throws IOException {
                ^
1 error
36 warnings

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':lib:javadoc'.
> Javadoc generation failed. Generated Javadoc options file (useful for troubleshooting): '/home/ligi/git/3rd/keycard/status-keycard-java/lib/build/tmp/javadoc/javadoc.options'

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 34s
127 actionable tasks: 123 executed, 4 up-to-date
```

--- a/demo-android/build.gradle
+++ b/demo-android/build.gradle
@@ -18,7 +18,7 @@ android {
     }
 
     lintOptions {
-        abortOnError false
+        lintConfig file("lint-config.xml")
     }
 
     compileOptions {

--- a/demo-android/lint-config.xml
+++ b/demo-android/lint-config.xml
@@ -1,0 +1,5 @@
+<lint>
+    <issue id="InvalidPackage">
+        <ignore path="**/bcprov-jdk15on-*.jar"/>
+    </issue>
+</lint>

